Docs/Help/Convert-JsonToBicep.md

```diff
@@ -57,13 +57,21 @@ Convert-JsonToBicep -String $json
 
 This example converts a JSON array to Bicep Language
 
-### Example 3: Read a file and convert to Bicep Language
+### Example 3: Read a file and convert it to Bicep Language
 ```powershell
 Get-Content -Path <path to .json file> -Raw | Convert-JsonToBicep
 ```
 
 This example converts a JSON file to Bicep Language
 
```

Comment:
Can you also update the syntax and parameters section in the docs with `-ToClipboard`. You can get it autogenerated using the `Update-MarkdownHelp` command.